servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/H2ServerParentConnectionContext.java

```diff
@@ -74,6 +74,11 @@ private H2ServerParentConnectionContext(final Channel channel, final BufferAlloc
         this.listenAddress = requireNonNull(listenAddress);
     }
 
+    @Override
+    public void acceptConnections(final boolean accept) {
+        channel().config().setAutoRead(accept);
```

Comment:
Does this prevent new connections from being put in the accept queue? What happens to connections in the accept queue? Are they rejected or will they just timeout on the client side?

Reply:
This stops the server socket from accepting a connection, the rest of the behavior observed is an indirect result.
Example:
- If the socket is not accepted, then new connections will be put in the accept-queue up-to the backlog allowance. These connections will appear as established on the client side, and if they have an idle timeout they will expire.
- If the accept queue is full, the SYN attempts will be backlogged, meaning that the client will repeat SYN attempts and the connection won't establish. In such cases if a connection timeout exists on the client side will be respected.